packages/driver/cypress/integration/commands/actions/type_events_spec.js

```diff
@@ -0,0 +1,710 @@
+const { _, $ } = Cypress
```

Comment:
Can we break these tests out in a different PR? I can't tell what tests were added for the changes you made.

Reply:
I temporarily removed `type_events_spec.js`. I'll bring it back in the next PR.